vj4/handler/problem.py

```diff
@@ -580,16 +580,13 @@ class ProblemCreateHandler(base.Handler):
   @base.post_argument
   @base.require_csrf_token
   @base.sanitize
-  async def post(self, *, title: str, content: str, hidden: bool=False, numeric_pid: bool=False):
-    pid = None
-    if numeric_pid:
-      pid = await domain.inc_pid_counter(self.domain_id)
+  async def post(self, *, title: str, content: str, hidden: bool=False, pname: str=None):
```

Comment:
This handler does not need to be changed, according to our discussion.
Add a `doc_name` field, and allow users to change it through ProblemSettingsHandler.
Further discussions needed.